docs/pages/foundations/animation/guidelines.js

```diff
@@ -0,0 +1,215 @@
+          title="Duration"
+          description="Animations should be direct and purposeful, meaning they only last as long as necessary to achieve the desired affect. For [accessibility reasons], avoid animations longer than 5 seconds. Most animations happen in a matter of milliseconds to avoid distracting the user. In most cases, aim to keep animations under one second. In the future, we will provide design tokens for desired durations in product."
+        />
+        <MainSection.Subsection
+          title="Easing and velocity"
+          description={`
+          The timing function of an animation should be determined by the property and object being animated.
+
+          Properties unrelated to motion (like color and opacity) will not likely benefit from easing and can be done linearly. For example, a background color of a button that expands or changes upon hover can happen linearly as it provides feedback to the user about their cursor position.
+
+          However, people experience motion non-linearly, so objects being moved can benefit from an easing timing function. For example, an object transitioning onto the screen from the right side may use ease-out as it slides into view to create visibility as fast as possible, and then ease-in upon its departure to create a smooth transition back to the main screen.
+
+          [Learn more about easing functions](https://www.smashingmagazine.com/2021/04/easing-functions-css-animations-transitions/#cubic-b%C3%A9zier-functions)
+          `}
+        >
+          <SandpackExample
+            code={easeExample}
+            name="Ease Example"
+            hideEditor
+            hideControls
+            previewHeight={800}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection
+        name="Accessibility"
+        description={`
+        Animations should never cause a user harm or distract from a necessary task. However, inaccessible animations have the potential to cause seizures, motion sickness, or anxiety in users, so we must ensure we are creating accessible animations every time.
+
+        Below are 3 key considerations for accessibility. Note: these do not apply for “essential” animations, where [essential is defined](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html#essentialdef) as any animation where “if removed, would fundamentally change the information or functionality of the content, _and_ information and functionality cannot be achieved in another way that would conform.”
+`}
+      >
+        <MainSection.Subsection
+          title="Looping animations"
+          description={`
+          Ensure that any animation that is moving, blinking, scrolling, or auto-playing and lasts longer than 5 seconds has an affordance for pausing and resuming the animation or an option to hide the animation.
+
+          _Looping animations should be avoided whenever possible, as they can become distracting very quickly._
```

Comment:
Should we not loop the above animation? 😅 (or provide a way to pause it?)

Reply:
hmm good point